docs/source/visualization/using_pycmc.ipynb

```diff
@@ -0,0 +1,146 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Visualizing Web Map Tile Service (WMTS) Layers"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Here is an example of how to visualize WMTS layers using pyCMC"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Import the ipycmc module\n",
+    "import ipycmc"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We utilize the Common Mapping Client (CMC) widget to visualize data in the MAAP Algorithm Development Environment (ADE)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "a263e522f3b24eb0b8591aced0a1e9a9",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "MapCMC()"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# utilize the CMC widget\n",
+    "w = ipycmc.MapCMC()\n",
+    "w"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The CMC user interface allows us to turn map layers on and off, manage their positioning and opacity, and zoom to layers using the Map Layers drop-down menu. There are also tools to allow us to zoom to the global extent, zoom in and out, switch between 2D and 3D maps, select areas, select basemaps, and toggle full screen mode. At the bottom left of the user interface, there is a text box allowing us to change the date and time and arrows to change the day. At the bottom right, the cursor location on the map is displayed in Latitude/Longitude decimal degrees."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Visualizing the Default Layers\n",
+    "\n",
+    "To visualize the default set of layers, we utilize the `load_layer_config()` method, handling the url as a WMTS xml file"
```

Comment:
Period or colon at the end of this sentence.

Reply:
Agreed and staged for next commit.